### ISSUE TITLE
New website

### ISSUE THREAD

**Issue by Dany0** [iamgreaser/iceball#125]
We need to look presentable for the Saturday event.

![Remove ponies](https://dl.dropboxusercontent.com/u/18131766/ShareX/2015/02/icebawl.png)

**Comment by Warp00** [comment]
Happy to help with this- websites are my day job.

Want my advice? Get bootstrap theme and/or Wordpress. Customise and add good pictures and content. You'll be done in a few hours.

**Comment by neppermint** [comment]
I wouldn't be terribly worried about about the website for the moment

**Comment by Dany0** [comment]
You wrote "about" twice.

**Comment by rakiru** [comment]
>Wordpress
HELL. FUCKING. NO.

Yes, hello, I also make websites for a living.

**Comment by Warp00** [comment]
@rakiru 

That's fair. I don't use WP much myself, and it's too bulky. However, It is easy to install. I was thinking of speed.

**Comment by rakiru** [comment]
Sure, it is incredibly fast at getting you hacked.

**Comment by Warp00** [comment]
@rakiru 

Yes, it is targeted a lot. However, if you use the appropriate precautions then it's not so bad. Anyway, what should be used for the new site?

**Comment by NotAFile** [comment]
@Warp00 I would recommend HTML and CSS. JS too if we absolutely need it. Kind of obscure, I know.

stupid jokes aside, I doubt we need any more than a static website, except for a forum, for which we can't choose the framework anyway. If we *really* need a dynamic website, I would vouch for Lua *cgi for simple stuff or whatever Lua framework is most popular if we need more than that.

**Comment by Warp00** [comment]
@NotAFile

http://grabaperch.com/ May be what is the best.

**Comment by NotAFile** [comment]
@Warp00 have a look at something like http://ultros.io/.

even a quarter of that site would be perfect, and could be implemented using only static pages. No need to pull in a whole CMS just for a simple website.

**Comment by Warp00** [comment]
@NotAFile Maybe. Anyway, not up to me. Perch isn't exactly a CMS in the conventional sense.
